perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -352,20 +358,15 @@ def Prepare(benchmark_spec):
 
   # Choose a disk or file name and optionally fill it
   disk = vm.scratch_disks[0]
-  mount_point = disk.mount_point
-
-  logging.info('Umount scratch disk on %s at %s', vm, mount_point)
-  vm.RemoteCommand('sudo umount %s' % mount_point)
 
   if FLAGS.against_device or FLAGS.prefill_device:
     logging.info('Fill device %s on %s', disk.GetDevicePath(), vm)
     FillDevice(vm, disk, FLAGS.device_fill_size)
 
-  if not FLAGS.against_device:
+  if not FLAGS.against_device and FLAGS.prefill_device:
```

Comment:
If we are not prefilling it, then it is already mounted (it is only not mounted if the against_device or prefill_device flags are specified).